main.tex

```diff
@@ -192,15 +192,16 @@ \subsubsection{Reductions}
 More reductions: If $A$ has an axis $\nset{foo}{n}$, then
 \begin{align*}
   \nsum{foo} A &= \sum_{i=1}^n A_{\nidx{foo}{i}} = \nmatrix{}{bar}{4 & 6 & 13} \\
+  \nfun{foo}{min} A &= \min \{A_{\nidx{foo}{i}} \mid 1 \leq i \leq n\} = \nmatrix{}{bar}{1 & 1 & 4} \\
+  \nfun{foo}{max} A &= \max \{A_{\nidx{foo}{i}} \mid 1 \leq i \leq n\} = \nmatrix{}{bar}{3 & 5 & 9} \\
   \nfun{foo}{norm} A &= \sqrt{\nsum{foo} A^2} = \nmatrix{}{bar}{\sqrt{10} & \sqrt{26} & \sqrt{97}} \\
-  \nfun{foo}{min} A &= \min_{i} A_{\nidx{foo}{i}} = \nmatrix{}{bar}{1 & 1 & 4} \\
-  \nfun{foo}{max} A &= \max_{i} A_{\nidx{foo}{i}} = \nmatrix{}{bar}{3 & 5 & 9} \\
-  \nfun{foo}{mean} A &= \frac{1}{n} A = \nmatrix{}{bar}{2 & 3 & 6.5} \\
+  \nfun{foo}{mean} A &= \frac{1}{n} \nsum{foo} A = \nmatrix{}{bar}{2 & 3 & 6.5} \\
   \nfun{foo}{var} A &= \frac{1}{n} \nsum{foo} (A - \nfun{foo}{mean} A)^2 = \nmatrix{}{bar}{1 & 4 & 6.25}
 \end{align*}
-(Note that $\max$ and $\min$ are overloaded; with a single argument and an axis subscript, they are reductions; otherwise, they are elementwise.)
+The summation, $\max$, and $\min$ operators are overloaded. If the operator is applied to a tensor and has an axis under it, then it's a reduction over that axis. But if it is applied to a set of tensors, it's an elementwise operation.
 
 You can also write multiple names to perform the reduction over multiple axes at once.
+But unlike in NumPy and its derivatives, a reduction with no axis under it is performed over zero axes (which in most cases does nothing).
```

Comment:
Hmm, is there ever a reason to do this? What if we just say that for `sum` you always need to have an axes? There is never a time in math when you would have a sum with no index (as far as I know)